index.html

```diff
@@ -81,6 +81,176 @@ <h2>Glad to see you!</h2>
+              </div>
+              <a href="#" class="work-btn">See Project</a>
+            </div>
+          </div>
+          <div class="card card-item-4">
+            <div class="card-work">
+              <img src="./images/art-printing.svg" alt="Art printing" />
+              <h2>Multi-Post Stories</h2>
+              <div class="card-caption">
+                <span>Canopy</span>
+                <img src="./images/Counter.svg" alt="counter" />
+                <span>Back End Dev</span>
+                <img src="./images/Counter.svg" alt="counter icon" />
+                <span>2015</span>
+              </div>
+              <div class="work-text">
+                <p>
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required.
+                </p>
+              </div>
+              <div class="stacks">
+                <ul>
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javascript</li>
+                </ul>
+              </div>
+              <a href="#" class="work-btn">See Project</a>
+            </div>
+          </div>
+        </section>
+      </div>
+      <div class="about-wrapper">
+        <section class="headline">
+          <div class="about">
+            <h2>About <span>Myself</span></h2>
+          </div>
+
+          <div class="about-desc">
+            <p>
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+
+          <div class="social">
+            <a href="#" class="about-btn">LET’S CONNECT</a>
+            <ul class="social-items">
+              <li>
+                <a href="#"
+                  ><img src="./images/Vector_twitter.png" alt="twitter profile"
+                /></a>
+              </li>
+
+              <li>
+                <a href="#"
+                  ><img src="./images/Linkedin_icon.png" alt="Linkedin profile"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./images/Vector_github.png" alt="github profile"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./images/Vector_medium.png" alt="Medium profile"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./images/Vector_ang.png" alt="ang profile"
+                /></a>
+              </li>
+            </ul>
+            <a href="#" class="resumeBtn">Get my resume</a>
+          </div>
+
+          <div class="languages">
+            <ul class="language-wrapper">
+              <li>Languages <i class="fas fa-chevron-right"> </i></li>
```

Comment:
- [ ] Following the figma design, there's a list of Languages(e.g., "JavasScript", "HTML" and "CSS") in the Language category please add it

Your design
![li1](https://user-images.githubusercontent.com/53356820/143625518-cc8f6e9c-c9f4-43eb-b259-88a4e8ce439a.png)

Figma design
![li](https://user-images.githubusercontent.com/53356820/143625469-974155b9-575d-435b-a400-5d82b072834a.png)

Reply:
Hi, @keneogu I believe the list will be populated when the drop-down is clicked is that really needed?